### PR TITLE
General channel

### PR DESCRIPTION
**Pull request by yummta**

### CODE REVIEW

File: app/src/App.js [hunk 1]
```diff
@@ -5,14 +5,35 @@ import Login from "./components/Login";
...
   React.useEffect(() => {
     if (user) {
       isSetLogin(true);
     }
   }, [user]);
 
-  return <>{isLogin ? <Chat user={user} /> : <Login setUser={setUser} />}</>;
+  return (
+    <>
+      {isLogin ? (
```
Comment (byteconde) [inline app/src/App.js]: Nice ternary operation, well done.

File: app/src/components/Chat.js [hunk 2]
```diff
@@ -17,8 +17,12 @@ function Chat({ user }) {
   return (
     <GridChat>
       <Profile user={user} />
-      <ChannelHeader />
-      <Channels />
+      <ChannelHeader currentChannel={currentChannel} />
```
Comment (AngieCristina) [inline app/src/components/Chat.js]: what exactly these lines (20 to 25) of code performs?

File: app/src/components/Channels.js [hunk 3]
```diff
@@ -1,10 +1,31 @@
 import React from "react";
-import ListChannels from "./ListChannels";
+import styled from "@emotion/styled";
 
-const channelList = ["general", "varios", "forobardo"];
+function Channels({ setCurrentChannel, channels, currentChannel }) {
```
Comment (byteconde) [inline app/src/components/Channels.js]: Can we reduce these 3 values into just one? 👀

File: app/src/components/Chat.js (hunk 2 above)
Comment (valevassallo) [reply app/src/components/Chat.js]: We're passing those that info to Chat's children, so that we can use them there

**Pull request merged**